### DISCUSSION TITLE
Kade Engine + another mod.

### DISCUSSION THREAD

**Discussion by ADreTega** [KadeArchive/Kade-Engine#27] [Q&A] [answered]
I've seen flippy do this where he mixed B-side and this engine together. I tried this out however the charts are B-side while the songs are are from the original FNF. Don't know how to mix these together.

**Comment by lilynx2509** [comment, marked as answer]
Kade himself combined the engine with B-Side, he sent it in his discord server in #kade-engine-resources . Here's the link in case you haven't seen it: https://drive.google.com/file/d/1mDNmySrJVBsTL8-NPJ5nPsDcVIaB4lf-/view

**Reply by piechy** [reply]
can you send me a link to the server please?
